Upstream: yes, http://code.google.com/p/uim/source/detail?r=1bb9041dd1068815316ec513135e2ed846eb1b99

commit 1bb9041dd1068815316ec513135e2ed846eb1b99
Author: nogu.dev <nogu.dev@gmail.com>
Date:   Sat Mar 10 10:40:56 2012 +0900

    * po/POTFILES.in
    * po/POTFILES.skip
      - Apply patch by dai to enable build with intltool 0.50.x (#47027).

diff --git a/po/POTFILES.in b/po/POTFILES.in
index a81eb79..ed1334c 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -28,6 +28,8 @@ scm/action.scm
 scm/ajax-ime-custom.scm
 scm/ajax-ime-key-custom.scm
 scm/ajax-ime.scm
+scm/annotation-filter.scm
+scm/annotation.scm
 scm/anthy-custom.scm
 scm/anthy-key-custom.scm
 scm/anthy-utf8-custom.scm
@@ -45,6 +47,7 @@ scm/elatin.scm
 scm/fileio.scm
 scm/generic-custom.scm
 scm/generic-key-custom.scm
+scm/generic-predict.scm
 scm/generic.scm
 scm/hangul.scm
 scm/http-client.scm
@@ -61,6 +64,7 @@ scm/look.scm
 scm/m17nlib.scm
 scm/mana-custom.scm
 scm/mana-key-custom.scm
+scm/ng-action.scm
 scm/predict-custom.scm
 scm/predict-sqlite3.scm
 scm/prime-custom.scm
diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index eb42e0d..ab9c9c0 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -1,4 +1,5 @@
 helper/GNOME_UimApplet.server.in
 qt4/pref/ui_keyeditformbase.h
 qt4/pref/ui_olisteditformbase.h
+test/test-custom.scm
 uim.desktop.in
